components/sidebar/sidebar_service.cc

```diff
@@ -34,6 +33,10 @@
 #include "ui/base/l10n/l10n_util.h"
 #include "ui/base/resource/resource_bundle.h"
 
+#if BUILDFLAG(ENABLE_PLAYLIST)
```

Comment:
I don't see `playlist/common/buildflags/buildflags.h` include here.

Reply:
Yeah, that was exactly what I ran into! thanks :)